README.md

```diff
@@ -11,9 +9,7 @@
 - Frameworks
 - Technologies used
```

Comment:
- [ ] Kindly, add the languages/technologies you used to create this project, such as HTML and CSS. 👌

> We should always be sure our repositories are documented professionally, please fix broken links on the readme file.